apps/ephemeral/specs/index.spec.tsx

```diff
@@ -13,4 +13,9 @@ describe('Index', () => {
     const { getByText } = render(<Index />);
     expect(getByText('Welcome to ephemeral!')).toBeTruthy();
   });
+
+  it('Resources & Tools should render on the page', () => {
```

Comment:
Do you not need to consider any asynchronous issues?
> `await wait()`...

Reply:
Not for static content rendered in a component.